### PR TITLE
remove outdated gtk package from code

### LINKED ISSUE #26563: Remove gtk imports from codebase
We have a few "import gtk" imports in the clipboards part of the code base but I cannot find any information as to what these actually are, so I assume these are especially outdated given our drop of Py27 support.

I *think* this used to be a part of PyGTK, but links to that (like http://www.pygtk.org/) seem to redirect now to PyGObject which has different import machinery anyway

### PR DESCRIPTION
**Pull request by xcz011**
- [x] closes #26563
- [x] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] whatsnew entry

### CODE REVIEW

**Review by WillAyd — CHANGES_REQUESTED** [review]
Generally looks good

File: doc/source/whatsnew/v0.25.0.rst [hunk 1]
```diff
@@ -598,6 +598,7 @@ Other
 - Removed unused C functions from vendored UltraJSON implementation (:issue:`26198`)
 - Bug in :func:`factorize` when passing an ``ExtensionArray`` with a custom ``na_sentinel`` (:issue:`25696`).
 - Allow :class:`Index` and :class:`RangeIndex` to be passed to numpy ``min`` and ``max`` functions.
+- Removed outdated packge gtk import for clipboard related (:issue:`26563`)
```
[the remark this answers is not in the document]
Comment (xcz011) [reply doc/source/whatsnew/v0.25.0.rst]: Got it, I will remove it soon

File: pandas/io/clipboard/__init__.py [hunk 2]
```diff
@@ -18,21 +18,20 @@
 On Linux, install xclip or xsel via package manager. For example, in Debian:
 sudo apt-get install xclip
 
-Otherwise on Linux, you will need the gtk, qtpy or PyQt modules installed.
+Otherwise on Linux, you will need the qtpy or PyQt modules installed.
 qtpy also requires a python-qt-bindings module: PyQt4, PyQt5, PySide, PySide2
 
-gtk and PyQt4 modules are not available for Python 3,
+PyQt4 modules are not available for Python 3,
```
[the remark this answers is not in the document]
Comment (xcz011) [reply pandas/io/clipboard/__init__.py]: By looking at PyQt4 document [https://www.riverbankcomputing.com/static/Docs/PyQt4/python_v3.html](url), they seems support python 3, so I am going remove this comment.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Lgtm @jreback

**Review by jreback — CHANGES_REQUESTED** [review]
can you add a note in the whatsnew (api changes) that we removed support for gtk for clipboards. ping on green.

**Comment by xcz011** [comment]
> can you add a note in the whatsnew (api changes) that we removed support for gtk for clipboards. ping on green.

Hi @jreback just added 😄

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @xcz011

**Pull request merged**